app/javascript/flavours/glitch/features/compose/components/compose_form.js

```diff
@@ -94,7 +94,7 @@ class ComposeForm extends ImmutablePureComponent {
     const { isSubmitting, isChangingUpload, isUploading, anyMedia } = this.props;
     const fulltext = this.getFulltextForCharacterCounting();
 
-    return !(isSubmitting || isUploading || isChangingUpload || length(fulltext) > maxChars || (!fulltext.trim().length && !anyMedia));
+    return !(isSubmitting || isUploading || isChangingUpload || length(fulltext) >= maxChars || (!fulltext.trim().length && !anyMedia));
```

Comment:
Why this change? If the full text length has reached exactly the max chars, it should be still considered acceptable, shouldn't it?

Reply:
yeah, you're right